packages/yoast/src/state/utils.ts

```diff
@@ -1,11 +1,28 @@
-import { YoastSocialDefaults, YoastMeta } from "../../types";
-import { Author, Taxonomy, PostType, Attachment } from "@frontity/source/";
+import Yoast, { YoastSocialDefaults, YoastMeta } from "../../types";
+import {
+  AuthorEntity,
+  TaxonomyEntity,
+  PostType,
+  AttachmentEntity,
+  Data
+} from "@frontity/source/types";
+import { State } from "frontity/types";
 
-export type Entity = (Author | Taxonomy | PostType | Attachment) & {
+export type Entity = (
+  | AuthorEntity
+  | TaxonomyEntity
+  | PostType
+  | AttachmentEntity) & {
   yoast_meta: YoastMeta;
 };
 
-export const getEntity = ({ state, data }): Entity => {
```

Comment:
Better to just assert the types of `state` and `data` and let the TS type inference do the magic 😅